lib/antora/extensions/antora-linked-worktree-patch.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+/* Copyright (c) 2022 OpenDevise, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License Version 2.0. If a copy of this license was not distributed
+ * with this file, you can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Again sorry I missed this in your initial posting, but given this is hosted in Spring Security's code base can we please get this updated to Apache 2 license header?

Reply:
Yep, sure thing.